PresentScreenings/Entities/Screening.cs

```diff
@@ -133,7 +133,15 @@ public Screening(string screeningText)
             }
 
             // Assign other properties.
-            Film = (from Film film in ScreeningsPlan.Films where film.FilmId == FilmId select film).First();
+            var filmList = ScreeningsPlan
+                .Films
+                .Where(f => f.FilmId == FilmId)
+                .ToList();
+            if (filmList.Count == 0)
+            {
+                AlertRaiser.QuitWithAlert("No film found for screening", $"Can't find film with ID={FilmId} while reading {Path.Combine(AppDelegate.DocumentsFolder, "screenings.csv")}.");
+            }
```

Comment:
Please consider to add a small text like "For developers:  If this is caused by a URL change of the same film, please replace the `filmId` in `ratings.csv`, `screenings.csv` (two columns) and `screeninginfo`. "

Reply:
Done.